Site/Controller/Factory.php

```diff
@@ -9,7 +9,7 @@
  */
 namespace Site\Controller;
 
-use chippyash\Type\String\StringType;
+use Chippyash\Type\String\StringType;
```

Comment:
Good spot.  Suspect that you are running on Mac or Windoze. My *nix machines are case insensitive for PHP.